docs/docs/use-cases/keda.md

```diff
@@ -0,0 +1,201 @@
+---
+comments: true
+---
+
+# Scaling Workloads with KEDA based on Keptn metrics
+
+If you want to use [KEDA](https://keda.sh/) to scale your workloads, you can set it up
+to consume metrics from Keptn.
+This gives you great flexibility for your setup since Keptn can
+consolidate different observability providers for you and KEDA
+will simplify the scaling operations for you.
+
+This use case is enabled by the Keptn Metrics
+[custom API](https://kubernetes.io/docs/reference/external-api/custom-metrics.v1beta2/)
+that the Keptn Metrics Operator provides.
+
+## Before you begin
+
+For this use case presentation, a few components need to be deployed to the
+cluster in order to have a full setup:
+
+- Keptn Metrics Operator: Will be used for metrics consolidation.
```

Comment:
Should this (or maybe the first mention of the Keptn Metrics Operator on the page) link to the "Components" page?  If I am a KEDA user who is new to Keptn, I have no idea what this is.

Reply:
done